bench/workgen/runner/runner/latency.py

```diff
@@ -80,9 +90,18 @@ def _latency_optype(fh, name, ch, t):
     print('  avg: ' + str(t.latency/t.latency_ops) + \
           ', min: ' + str(t.min_latency) + ', max: ' + str(t.max_latency),
           file=fh)
+
     us = t.us()
     ms = t.ms()
     sec = t.sec()
+    print('', file=fh)
+    _latency_buckets(fh, name + ' us count ', 1, us, 1000)
+    _latency_buckets(fh, name + ' ms count ', 1000, ms, 1000)
+    _latency_buckets(fh, name + ' sec count ', 1000000, sec, 100)
```

Comment:
WDYT about moving integral values (1,100,1000,1000000) to variables with meaningful names since we later could reuse them for `_show_buckets`